htdocs/index.php

```diff
@@ -155,7 +155,7 @@
 	require_once DOL_DOCUMENT_ROOT.'/core/class/workboardresponse.class.php';
 
 	// Number of actions to do (late)
-	if (isModEnabled('agenda') && empty($conf->global->MAIN_DISABLE_BLOCK_AGENDA) && $user->rights->agenda->myactions->read) {
+	if (isModEnabled('agenda') && empty($conf->global->MAIN_DISABLE_BLOCK_AGENDA) && !empty($user->hasRight('agenda', 'myactions', 'read'))) {
```

Comment:
If you use hasRight, you must not use empty
Use if $user->hasRight
or if ! $user->hasRight

Reply:
@eldy ok for me it's done